sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Demo/Azure.Monitor.OpenTelemetry.Exporter.Demo.csproj

```diff
@@ -5,6 +5,16 @@
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Do we need both of the entries here?

Reply:
good catch, thanks!
i'll clean this up!